src/tests/test_cases.py

```diff
@@ -90,14 +90,14 @@ def test_equality():
     s2.update(1, 2, 3, 4, 5, 6, 7)
 
     assert s == s2
-    s2.remove(1)
-    s2.add(1)
+    assert 0 == s2.remove(1)
+    assert 6 == s2.add(1)
```

Comment:
As I mentioned above, we should not validate return value of `add` method there as well.